stripe_pipeline/cli.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import click
+from stripe_pipeline.crawler import crawler
+
+
+@click.group()
+def cli():
+    pass
+
+
+cli.add_command(crawler)
```

Comment:
Write now the cli just has a single `crawler` command, but I plan on adding more commands, for things like initializing the database and possible add-hoc tasks

Reply:
Thanks @davidgasquez! That worked great, added it in [this commit](https://github.com/bufferapp/stripe-pipeline/pull/1/commits/c97db2db982acf4a64169b45b4903aa64db68323)